### PR TITLE
Rebranding in the Fostive style

### PR DESCRIPTION
**Pull request by DianaPerkinsDesign**
New emojis and colors!

Screen Shot 2020-09-17 at 9 40 23 PM

### CODE REVIEW

File: src/modules/emote/widget/widget.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1>Share how you feel!</h1>
                     <a href="#" class="text-link" onclick={openOptions}
                         >options</a
                     >
-                    <aside>Brought to you with ♡ by Heroku</aside>
+                    <aside>Brought to you with ♡ by Fostive</aside>
```
Comment (superhighfives) [inline src/modules/emote/widget/widget.html:21]: Should this link to the Fostive org, @DianaPerkinsDesign @justjenu?

**Comment by justjenu** [comment]
@crcastle @julianduque look good?

**Comment by superhighfives** [comment]
Just pinging on the above, @crcastle @julianduque.

File: src/modules/emote/widget/widget.html (hunk 1 above)
Comment (crcastle) [reply src/modules/emote/widget/widget.html:21]: Yes I think so, @superhighfives. I'll update code.

File: src/modules/emote/widget/widget.html [hunk 2]
```diff
@@ -18,7 +18,12 @@ <h1>Share how you feel!</h1>
                     <a href="#" class="text-link" onclick={openOptions}
                         >options</a
                     >
-                    <aside>Brought to you with ♡ by Heroku</aside>
+                    <aside>
+                        Brought to you with ♡ by&nbsp;
+                        <a target="_blank" href="https://github.com/fostive"
```
Comment (crcastle) [inline src/modules/emote/widget/widget.html:23]: @DianaPerkinsDesign or @superhighfives can I get your design-focused eyes on this quickly? If it looks good to you, please click merge on this PR!

Comment (superhighfives) [reply src/modules/emote/widget/widget.html:23]: Looks good to me! Approved. 👍

**Review by superhighfives — APPROVED** [review]
LGTM! 👍

**Pull request merged**